### PR TITLE
Note on `no-local-hooks`

### PR DESCRIPTION
**Pull request by pzeballos**
If you enable `no-local-hooks`, it will also affect plugins.

local-hook
config

### CODE REVIEW

**Review by plaindocs — COMMENTED** [review]
"Hosted" plugins feels confusing to me. 

> Non-Buildkite plugins hosted on GitHub require you to include the organization name as well as the plugin name, for example:

But they are still downloaded and run, they aren't "run in the cloud".  So maybe 

> Disabling local hooks also disables all plugins.

What do you think?

**Comment by pzeballos** [comment]
For me is clear. A user read: "Disabling local hooks will also disable plugins" and asked "Only repository plugins or no plugins in general?". But maybe adding the `all` it will make it clearer

**Comment by plaindocs** [comment]
Yeah, I guess my main objection is referring to them as "hosted plugins". If we want to call them out specifically we could do something like

> Disabling local hooks also disables plugins from all sources.

or 

> Disabling local hooks also disables plugins from all sources, including local plugins and plugins hosted on GitHub.

I'd tend towards the first option.

**Comment by pzeballos** [comment]
I like the first option! :)

File: data/agent_attributes.yaml [hunk 1]
```diff
@@ -192,7 +192,7 @@ attributes:
   default_value: "false"
   required: false
   desc: | 
-      Do not execute any local hooks.
+      Do not execute any local hooks, or plugins from any sources..
```
Comment (plaindocs) [inline data/agent_attributes.yaml:195]:
```suggestion
      Do not execute any local hooks, or plugins from any source.
```

**Review by plaindocs — APPROVED** [review]
LGTM

**Pull request merged**